src/components/books.css

```diff
@@ -1,14 +1,137 @@
-.book {
-  border: 1px black solid;
+.books {
+  display: flex;
+  width: 100%;
+  justify-content: space-around;
+  align-items: center;
+  padding: 1.688rem;
+  background-color: #fff;
+  border-radius: 2%;
+  border: 0.8px solid #e8e8e8;
+  margin-top: 24px;
+  min-height: 50px;
 }
 
-.buttons {
-  display: flex;
+.category {
```

Comment:
- [ ] Your styling is almost complete, but the fonts are still missing, please check on Zeplin the corresponding font for every element, click on any text to check it:

<img width="936" alt="Screen Shot 2022-07-16 at 11 34 53 PM" src="https://user-images.githubusercontent.com/83717285/179384172-e1baf563-40d6-493e-a368-17752d90a6a7.png">